README.md

```diff
@@ -8,9 +8,110 @@ Ubuntu Proxy Manager is a D-Bus mediated service that allows for managing system
 [![Go Report Card](https://goreportcard.com/badge/ubuntu/ubuntu-proxy-manager)](https://goreportcard.com/report/ubuntu/ubuntu-proxy-manager)
 [![License](https://img.shields.io/badge/License-GPL3.0-blue.svg)](https://github.com/ubuntu/ubuntu-proxy-manager/blob/main/LICENSE)
 
+## Installation
+
+Use APT to install the package on Ubuntu, which will set up all the required configuration files:
+
+```sh
+sudo apt install ubuntu-proxy-manager
+```
+
+## Usage
+
+The service currently exposes a single D-Bus method, `com.ubuntu.ProxyManager.Apply`, taking 6 string arguments:
+- `http` - HTTP proxy
+- `https` - HTTPS proxy
+- `ftp` - FTP proxy
+- `socks` - SOCKS proxy
+- `no_proxy` - hosts excluded from proxy
+- `auto` - proxy autoconfiguration URL
+
+When calling the function, all 6 arguments must be passsed. Arguments can be skipped by replacing them with empty strings. Keep in mind that this function is not additive and it replaces previously set proxy settings on each call.
+
+``` sh
```

Comment:
Maybe add an example (with some generic values or wildcards) to apply all proxy settings? I think it will be better to help people understand the order of the arguments.